src/main/scala/com/gu/ami/amiup/UI.scala

```diff
@@ -48,6 +53,27 @@ object UI extends LazyLogging {
     println("-----------------------".colour(Console.RESET))
   }
 
+  def displayRefreshProgress(refreshes: Seq[InstanceRefreshProgress]): Unit = {
+    refreshes.foreach {
+      case InstanceRefreshProgress(asgName, true, true, _, _, _) =>
+        // Instance refresh has been cancelled or has failed
+        println(s"$asgName: Instance refresh was not successful".colour(Console.RED))
+      case InstanceRefreshProgress(asgName, false, _, _, _, _) =>
+        println(s"$asgName: Starting instance refresh...".colour(Console.BLUE))
+      case InstanceRefreshProgress(asgName, true, false, true, status, statusReason) =>
+        println(s"$asgName: ${parseStatus(status, statusReason)}".colour(Console.GREEN))
+      case InstanceRefreshProgress(asgName, true, false, _, status, statusReason) =>
+        println(s"$asgName: ${parseStatus(status, statusReason)}".colour(Console.CYAN))
+    }
+    println("-----------------------".colour(Console.RESET))
```

Comment:
`RESET` shouldn't be needed here. In the definition [below](https://github.com/guardian/amiup/pull/3/files#diff-81bc6c1699365595111f2a4253bd5641755f009a103ec2e255b643164f345918R77) you'll see that it always resets at the end of every string.